extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -14,53 +14,91 @@
  * limitations under the License.
  */
 
+import {CONSENT_ITEM_STATE, ConsentStateManager} from './consent-state-manager';
 import {CSS} from '../../../build/amp-consent-0.1.css';
 import {ConsentPolicyManager} from './consent-policy-manager';
-import {ConsentStateManager} from './consent-state-manager';
 import {Layout} from '../../../src/layout';
+import {
+  NOTIFICATION_UI_MANAGER,
+  NotificationUiManager,
+} from '../../../src/service/notification-ui-manager';
 import {Services} from '../../../src/services';
 import {assertHttpsUrl} from '../../../src/url';
 import {
   childElementsByTag,
   isJsonScriptTag,
 } from '../../../src/dom';
-import {dict} from '../../../src/utils/object';
+import {dev, user} from '../../../src/log';
+import {dict, map} from '../../../src/utils/object';
 import {getServicePromiseForDoc} from '../../../src/service';
 import {isExperimentOn} from '../../../src/experiments';
 import {parseJson} from '../../../src/json';
-import {user} from '../../../src/log';
+import {setStyle, toggle} from '../../../src/style';
 
 const CONSENT_STATE_MANAGER = 'consentStateManager';
 const CONSENT_POLICY_MANGER = 'consentPolicyManager';
 const AMP_CONSENT_EXPERIMENT = 'amp-consent';
 const TAG = 'amp-consent';
 
+/**
+ * @enum {boolean}
+ */
+const ACTION_TYPE = {
+  // We can support DISMISS if requested
+  ACCEPT: true,
+  REJECT: false,
+};
+
 
 export class AmpConsent extends AMP.BaseElement {
   constructor(element) {
     super(element);
 
-    /** @private {?./consent-state-manager.ConsentStateManager} */
+    /** @private {?ConsentStateManager} */
     this.consentStateManager_ = null;
 
-    /** @private {?./consent-policy-manager.ConsentPolicyManager} */
+    /** @private {?ConsentPolicyManager} */
     this.consentPolicyManager_ = null;
 
+    /** @private {?NotificationUiManager} */
+    this.notificationUiManager_ = null;
+
+    /** @private {!Object<string, Element>} */
```

Comment:
can you add ! or ? here to be sure I know intent of nullability

Reply:
done